DANS-KNAW/dd-verify-migration
# DD-784: load-from-fedora/vault: add File rights

Fixes DD-784: load-from-fedora/vault: add File rights

# Description of changes
* read file rights from files.xml, 
* when not found, apply defaults from dataset.xml
  * [x] for load-from-vault
  * [x] for load-from-fedora, the columns `accessible_to` and  `visible_to` of `easy_files` are defined not-null. So it seems we don't need to read `dataset.xml`.
* new formatting for some classes lead to a lot of white space changes


# How to test

build, deploy and run on deasy: copy `src/test/resources/deasy.csv` to `easy-dtap/shared` and:

    dd-verify-file-migration load-from-fedora /vagrant/shared/deasy.csv



# Related PRs
(Add links)
*

# Notify
@DANS-KNAW/dataversedans
